gulp-html-replace/gulp-html-replace.d.ts

```diff
@@ -0,0 +1,29 @@
+// Type definitions for gulp-html-replace v1.5.5
+// Project: https://www.npmjs.com/package/gulp-html-replace
+// Definitions by: Peter Juras <https://github.com/peterjuras>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+/// <reference path="../node/node.d.ts" />
+
+declare module "gulp-html-replace" {
+  interface IAdvancedTask {
```

Comment:
http://www.typescriptlang.org/Handbook#writing-dts-files

> Naming Conventions
In general, do not prefix interfaces with I (e.g. IColor). Because the concept of an interface in TypeScript is much more broad than in C# or Java, the IFoo naming convention is not broadly useful.